_topic_maps/_topic_map.yml

```diff
@@ -3211,38 +3211,19 @@ Topics:
     File: cnf-image-based-upgrade-base
   - Name: Performing an image-based upgrade for single-node OpenShift clusters using GitOps ZTP
     File: ztp-image-based-upgrade
-# Remove RDS from published docs until 4.16 RDS is released
-# ---
-# Name: Reference design specifications
-# Dir: telco_ref_design_specs
-# Distros: openshift-telco
-# Topics:
-# - Name: Telco reference design specifications
-#   File: telco-ref-design-specs-overview
-# - Name: Telco RAN DU reference design specification
-#   Dir: ran
-#   Topics:
-#   - Name: Telco RAN DU reference design overview
-#     File: telco-ran-ref-design-spec
-#   - Name: Telco RAN DU use model overview
-#     File: telco-ran-du-overview
-#   - Name: RAN DU reference design components
-#     File: telco-ran-ref-du-components
-#   - Name: RAN DU reference design configuration CRs
-#     File: telco-ran-ref-du-crs
-#   - Name: Telco RAN DU software specifications
-#     File: telco-ran-ref-software-artifacts
-# - Name: Telco core reference design specification
-#   Dir: core
-#   Topics:
-#   - Name: Telco core reference design overview
-#     File: telco-core-rds-overview
-#   - Name: Telco core use model overview
-#     File: telco-core-rds-use-cases
-#   - Name: Core reference design components
-#     File: telco-core-ref-design-components
-#   - Name: Core reference design configuration CRs
-#     File: telco-core-ref-crs
+- Name: Image-based installation for single-node OpenShift clusters
+  Dir: image-base-install
+  Topics:
+  - Name: Understanding an image-based installation and deployment for single-node OpenShift clusters
+    File: ibi-understanding-image-based-install
+  - Name: Preparing for an image-based installation for single-node OpenShift clusters
+    File: ibi-preparing-for-image-based-install
+  - Name: Performing an image-based installation for single-node OpenShift clusters
```

Comment:
These 2 titles are too similar consider rephrasing

Reply:
Reworked the titles to the following:

- Understanding an image-based installation and deployment for single-node OpenShift clusters
- Preparing for an image-based installation for single-node OpenShift clusters
- Installing a single-node OpenShift cluster using an image-based installation
- Deploying a managed single-node OpenShift cluster using an image-based deployment
- Deploying a standalone single-node OpenShift clusters using an image-based deployment